tests/src/common/WskTestHelpers.scala

```diff
@@ -118,3 +118,22 @@ trait WskTestHelpers extends Matchers {
         }
     }
 }
+
+object JsObjectExtensions {
```

Comment:
Done. Moved code to a `JsHelpers` trait in a separate file. And it even works!

Thanks @psuter!